src/util/OGCFilter.js

```diff
@@ -0,0 +1,328 @@
+/* Copyright (c) 2015-2019 The Open Source Geospatial Foundation
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/**
+ * A utility class for converting ExtJS filters to OGC compliant filters
+ *
+ * @class GeoExt.util.OGCFilter
+ */
+Ext.define('GeoExt.util.OGCFilter', {
```

Comment:
Maybe `util` is a bit underestimation. I think this could go to an own package, like `filter`. But I am OK with `util` of course

Reply:
Will keep it as is